docs/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/migrate-back-end.md

```diff
@@ -10,44 +10,54 @@ canonicalUrl: http://docs.strapi.io/developer-docs/latest/update-migration-guide
 
 Migrating the back end of a plugin to Strapi v4 requires:
 
-- updating [imports](#updating-imports)
+- updating [Strapi packages](#updating-strapi-packages)
 - updating content-types [getters](#updating-content-types-getters) and, optionally, [relations](#updating-content-types-relations)
 - updating the [plugin configuration](#updating-plugin-configuration)
 
 Depending on these steps, some actions can only be done manually while others can be performed automatically by scripts that modify the code, which are called codemods. The following table lists available options for each step of the migration:
 
 | Action                         | Migration type                                                                                               |
 | ------------------------------ | ------------------------------------------------------------------------------------------------------------ |
-| Update imports                 | [Automatic](#automatic-imports-update) or [manual](#manual-imports-update)                             |
+| Update Strapi packages                 | [Automatic](#automatic-strapi-packages-update) or [manual](#manual-strapi-packages-update)                             |
 | Update content-types getters   | [Automatic](#automatic-content-types-getters-update) or [manual](#manual-content-types-getters-update) |
 | Update content-types relations | [Manual](#updating-content-types-relations)                                                                    |
 | Update configuration           | [Manual](#updating-plugin-configuration)                                                                              |
 
-## Updating imports
+## Updating Strapi packages
 
 :::strapi v3/v4 comparison
 Package names in Strapi v3 are prefixed by `strapi-`.
 
-Strapi v4 uses scoped imports.
+Strapi v4 uses scoped packages.
 :::
 
-To migrate to Strapi v4, update all Strapi imports from `strapi-package-name` to `@strapi/package-name`. Imports can be updated [automatically](#automatic-imports-update) or [manually](#manual-imports-update).
+To migrate to Strapi v4, rename all Strapi packages from `strapi-package-name` to `@strapi/package-name`. You will need to do this in the `package.json` dependencies and anywhere the package is imported.
```

Comment:
```suggestion
To migrate to Strapi v4, rename all Strapi packages from `strapi-package-name` to `@strapi/package-name`. This needs to be done in the `package.json` dependencies and anywhere the package is imported.
```

Reply:
(to comply with our https://handbook.strapi.io/user-success-manual/12-rules-of-technical-writing, esp. usage of "you")